papermario/src/code_ebd0_len_6a0.c

```diff
@@ -1,15 +1,59 @@
 #include "common.h"
 
-INCLUDE_ASM(code_ebd0_len_6a0, func_800337D0);
+void func_800337D0(s16 new_fade_status) {
```

Comment:
if/when you use the r/g/b/alpha in the struct, this should probably become "new_alpha" as the argument